fusedwind/turbine/test/data/DTU10MW.dp3d

```diff
@@ -1,4 +1,5 @@
 # web00  web01  web02  web03
+# top  mid  mid  mid
```

Comment:
don't know if it should be the same as the release version, since we'd then have to update this version every time we make a new release. so maybe just an integer 

    # version 1

or does it make sense to have a sub version, i.e. `# version 1 0`? and yes, this should be added to all files. we should still be able to read files without the version string, so the code should read the first line, check if it contains "version" and if not assume it to be version 1 and continue. the st3d dict should then also have an entry with `version`.